src/components/dialogs/parameters/parameters.js

```diff
@@ -0,0 +1,258 @@
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const [tabIndex, setTabIndex] = useState(0);
+
+    const studyUuid = useSelector((state) => state.studyUuid);
+
+    function TabPanel(props) {
+        const { children, value, index, ...other } = props;
+        return (
+            <Typography
+                component="div"
+                role="tabpanel"
+                hidden={value !== index}
+                id={`simple-tabpanel-${index}`}
+                aria-labelledby={`simple-tab-${index}`}
+                {...other}
+            >
+                {value === index && <Box p={1}>{children}</Box>}
+            </Typography>
+        );
+    }
+
+    const resetNetworkParameters = () => {
+        fetchDefaultParametersValues().then((defaultValues) => {
+            const defaultFluxConvention = defaultValues.fluxConvention;
+            if (
+                Object.values(FluxConventions).includes(defaultFluxConvention)
+            ) {
+                this.props.NetworkParameters.handleChangeFluxConvention(
+                    defaultFluxConvention
+                );
+            }
+        });
+    };
+
+    return (
+        <Dialog
+            open={showParameters}
+            onClose={hideParameters}
+            aria-labelledby="form-dialog-title"
+            maxWidth={'md'}
+            fullWidth={true}
+        >
+            <DialogTitle id="form-dialog-title">
+                <Typography
+                    component="span"
+                    variant="h5"
+                    className={classes.title}
+                >
+                    <FormattedMessage id="parameters" />
+                </Typography>
+            </DialogTitle>
+            <DialogContent>
+                <Container maxWidth="md">
+                    <Tabs
+                        value={tabIndex}
+                        variant="scrollable"
+                        onChange={(event, newValue) => setTabIndex(newValue)}
+                        aria-label="parameters"
+                    >
+                        <Tab
+                            label={<FormattedMessage id="SingleLineDiagram" />}
+                        />
+                        <Tab label={<FormattedMessage id="Map" />} />
+                        <Tab
+                            disabled={!studyUuid}
+                            label={<FormattedMessage id="LoadFlow" />}
+                        />
+                        <Tab label={<FormattedMessage id="Network" />} />
+                    </Tabs>
+
+                    <TabPanel value={tabIndex} index={0}>
+                        <SingleLineDiagramParameters />
+                    </TabPanel>
+                    <TabPanel value={tabIndex} index={1}>
+                        <MapParameters />
+                    </TabPanel>
+                    <TabPanel value={tabIndex} index={2}>
+                        {studyUuid && (
+                            <LoadFlow hideParameters={hideParameters} />
```

Comment:
There is an annoying growing visal effect when I move from the Map tab to the Loadflow one. For half a second, the dialog is small, then immediately grows to full size.